client/src/app/dashboard/crawler/crawler.component.css

```diff
@@ -0,0 +1,13 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  margin-top: 20px;
+}
+
+.button-row button,
+.button-row a {
+  margin-right: 8px;
```

Comment:
I believe it would be better to use em instead of px, since this is a relative unit dependent of the used font and its size. (https://www.w3schools.com/cssref/css_units.asp)